src/Java.md

```diff
@@ -34,6 +34,7 @@ It would be appreciated if there are any Java VTK experts who could convert any
 | Example Name | Classes Demonstrated | Description | Image |
 | -------------- | ---------------------- | ------------- | ------- |
 [Delaunay2D](/Java/Filtering/Delaunay2D) | vtkDelaunay2D | An example to generate set of points and to triangulate using Delaunay2D
+[Glyph2D](/Java/Filtering/Glyph2D) | vtkGlyph2D | Example to Copy a polydata to every point in the input set. 
 [Glyph3D](/Java/Filtering/Glyph3D) | vtkGlyph3D | This example applies an object at every point using vtkGlyph3D
```

Comment:
Copy paste error here for Glyph2d.

Reply:
I have deleted the whole line and typed in again. Please check now.